plugin/code_actions.py

```diff
@@ -63,9 +67,10 @@ def request_for_region_async(
                     return task
                 else:
                     self._response_cache = None
-        elif only_kinds == [CodeActionKind.Refactor]:
-            self.refactor_actions_key = "{}#{}:{}".format(view.buffer_id(), view.change_count(), region)
+        elif only_kinds == MENU_ACTIONS_KINDS:
```

Comment:
Python will not handle it correctly when both lists have the same items but in different order but I guess you are assuming that that's OK because we expect to only care about our specific request that we make?

Reply:
If I understand it correctly, you mean for example that `[1, 2] != [2, 1]`, but that should be irrelevant here, because we always refer to the `MENU_ACTIONS_KINDS` variable now, both when passing `only_kinds` to the function for the request, and then on this line when comparing it.